### PR TITLE
Refactor the log when failing to get hadoop fs delegation token

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

Refactor the log when failing to get hadoop fs delegation token 

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by turboFei** [comment]
also cc @zhouyifan279

File: kyuubi-server/src/main/scala/org/apache/kyuubi/credentials/HadoopFsDelegationTokenProvider.scala [hunk 1]
```diff
@@ -63,19 +63,26 @@ class HadoopFsDelegationTokenProvider extends HadoopDelegationTokenProvider with
 
   override def obtainDelegationTokens(owner: String, creds: Credentials): Unit = {
     doAsProxyUser(owner) {
-      val fileSystems = fsUris.map(FileSystem.get(_, hadoopConf)).toSet
+      val fileSystems = fsUris.map { uri =>
+        uri -> FileSystem.get(uri, hadoopConf)
+      }
```
Comment (pan3793) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/credentials/HadoopFsDelegationTokenProvider.scala:68]: This changes the semantics, omitting "dedup"

Comment (turboFei) [reply kyuubi-server/src/main/scala/org/apache/kyuubi/credentials/HadoopFsDelegationTokenProvider.scala]: thanks

**Review by pan3793 — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merging to master

**Comment by zhouyifan279** [comment]
Sorry for late reply. LGTM.

**Pull request closed without merging**